integration_tests/test_sage_basic_identification.py

```diff
@@ -121,3 +125,89 @@ def test_create_asset_group_identification(session, codex_url, test_root, login)
     )
     id_job_data = zebra_sighting.json()['jobs'][0]
     assert not id_job_data['active']
+
+
+def test_identification_international(session, codex_url, test_root, login):
+    login(session)
+    zebra_guids = create_sighting(session, codex_url, test_root, 'zebra.jpg')
+
+    # the first one does not go for identification, so make it processed so that the next one can
+    patch_response = session.post(
+        codex_url(f"/api/v1/sightings/{zebra_guids['sighting']}/reviewed"),
+        json=[],
+    )
+    assert patch_response.status_code == 200
+    description = 'International names Þröstur Sélène cédric characters &%$* ¼ ©,® ™ m².'
+    location = 'Montréal'
+
+    # Going to get nasty now and put international characters in here
+    filename = 'zebra-named-Sigurður.jpg'
+    group_data = {
+        'description': description,
+        'uploadType': 'form',
+        'speciesDetectionModel': ['african_terrestrial'],
+        'sightings': [
+            {
+                'time': '2000-01-01T01:01:01+00:00',
+                'timeSpecificity': 'time',
+                'locationId': 'Tiddleywink',
+                'verbatimLocation': location,
+                'encounters': [
+                    {
+                        'verbatimLocation': location,
+                    }
+                ],
+                'assetReferences': [filename],
+                'idConfigs': [
+                    {
+                        'matchingSetDataOwners': 'all',
+                        'algorithms': ['hotspotter_nosv'],
+                    }
+                ],
+            },
+        ],
+    }
+    zebra2_guids = create_sighting(session, codex_url, test_root, filename, group_data)
```

Comment:
Is the filename returned in any APIs?  I was wondering whether the filename changed but I can't see if any asserts in this test check for the asset filename.

Reply:
Yes. I'm not sure that it is tested specifically in the unit tests but it is tested in the integration tests:
https://github.com/WildMeOrg/houston/blob/main/integration_tests/test_asset_group_sightings.py#L184
And the filename returned is the one the user sent, not the one that the file is stored as.

Reply:
But it looks like the integration test filename doesn't have any special characters so it's just stored without modifications.

Reply:
I think you are assuming that only filenames with international characters are modified. That is not the case. 
With this change ALL Filenames are stored as the hash of the filename plus extension. That was what was decided at the Knucklehead on Wednesday.

Reply:
Oh yeah, `secure_filename` is completely removed :+1: